### DISCUSSION TITLE
Метод-геттер для activeActions в Resource

### DISCUSSION THREAD

**Discussion by twent** [moonshine-software/moonshine#375] [Ideas]
Приветствую 😊
Думаю, что было бы полезно иметь метод-геттер для activeActions.
Например, можно переопределить доступные действия в зависимости от текущего статуса или от какой либо другой логики.

**Comment by lee-to** [comment]
Есть такое 


<img width="582" alt="Screenshot 2023-07-08 at 15 58 44" src="https://github.com/moonshine-software/moonshine/assets/1861327/03db76a9-0944-41e0-806b-9a1e4c585d81">
